spec/models/user_spec.rb

```diff
@@ -21,25 +22,76 @@
     it { is_expected.to have_many(:rosters).through(:patrol) }
     it { is_expected.to have_many(:proficiency_signups) }
     it { is_expected.to have_many(:proficiencies).through(:proficiency_signups) }
+    it { is_expected.to have_many(:outreach_patrol_sign_ups) }
+    it { is_expected.to have_many(:outreach_patrol).through(:outreach_patrol_sign_ups) }
     it { is_expected.to have_many(:swaps) }
     it { is_expected.to have_many(:notices) }
     it { is_expected.to have_many(:notice_acknowledgements) }
   end
 
-  context '::name' do
-    let(:user) { create :user, first_name: 'First', last_name: 'Last' }
+  describe 'scopes' do
+    describe 'with_club' do
+      subject { described_class.with_club(club) }
+
+      let(:club) { create(:club) }
+
+      before { club }
 
-    it 'returns the correct name string' do
-      expect(user.name).to eq("#{user.first_name} #{user.last_name}")
+      it { is_expected.to be_empty }
     end
   end
 
-  describe User do
-    let(:club) { String(name: 'organisation') }
+  describe 'instance methods' do
+    describe 'default_position' do
```

Comment:
We do need to remove default position from user. 

This lives in the PatrolMember model instead. There is an open issue to permit a user to belong to >1 patrol (and >1 club too)

Leave for now and we'll do a separate or for fixing